x-pack/plugins/serverless_search/public/application/components/pipeline_overview.tsx

```diff
@@ -0,0 +1,47 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React from 'react';
+
+import {
+  EuiSpacer,
+  EuiText,
+  EuiButton,
+} from '@elastic/eui';
+import { i18n } from '@kbn/i18n';
+import { ApplicationStart } from '@kbn/core-application-browser';
+import { HttpStart } from '@kbn/core-http-browser';
+
+export interface PipelineOverviewProps {
```

Comment:
We should be grabbing these from the hook that's used in the parent property, rather than passing them down as props. In general we try not to pass down very complicated, composite objects as props. 

An alternative would be to construct the path in the parent property and pass that down as a prop.

Reply:
Yes, that makes sense. Should not sent complex object as props.